external/ag-website-shared/src/components/license-pricing/LicensePricing.tsx

```diff
@@ -206,20 +206,23 @@ export const LicensePricing: FunctionComponent<Props> = ({ defaultSelection }) =
 
                         <div className={styles.trialLicence}>
                             <div className={styles.trialLicenceCopy}>
-                                <h3 className="text-2xl">
+                                <h3
+                                    className={classnames(styles.trialLicenceHeader, 'text-2xl')}
+                                    id="request-trial-licence"
+                                >
                                     <Icon name="enterprise" svgClasses={styles.enterpriseIcon} />
-                                    <span>Enterprise Bundle Trial</span>
+                                    <p style={{ maxWidth: '16ch' }}>Start Your 30-Day Enterprise Bundle Trial</p>
                                 </h3>
 
-                                <p>
-                                    See the differences between our community and enterprise versions and request a
-                                    trial license
+                                <p style={{ maxWidth: '48ch' }}>
+                                    Explore the full enterprise capabilities of AG Grid and AG Charts with a free 30-day
+                                    trial licence — no restrictions, no watermarks.
                                 </p>
 
                                 <a
                                     id="request-trial-licence"
                                     className={classnames('button', styles.trialButton)}
-                                    href={gridUrlWithPrefix({
+                                    href={chartsUrlWithPrefix({
```

Comment:
Should probably be `urlWithPrefix` so it's relative to library website

```suggestion
                                    href={urlWithPrefix({
```

Reply:
Sorted in separate commit 👍